src/next-app/pages/invite.tsx

```diff
@@ -7,7 +7,7 @@ import styles from '../styles/pages/Invite.module.css';
 const InvitePage: NextPage = () => {
   return (
     <div className={styles.container}>
-      <Header goBackToHref="/create-account" title="Invite Employees" />
+      <Header goBackToHref="/create-shop" title="Invite Employees" />
```

Comment:
Why do we need this `goBackToHref`? Can't we do something like `router.back()`?

There is also an issue with this:
When you login as ShopOwner, then use the hamburger menu to select "Employees", you get taken to the Invite Employee Page. Then, if you use this back button, you get taken to "Create Shop". If you click back one more time, you get taken to "Create Account" page.

Reply:
Resolved. Changed to `router.back()` as recommended.